HMD/UserSettings/Layouts/default-2022.dwlt

```diff
@@ -187,8 +187,8 @@ MonoBehaviour:
     serializedVersion: 2
     x: 0
     y: 0
-    width: 288
-    height: 510.4
+    width: 287.2
+    height: 509.6
```

Comment:
there's a lot of these random numerical changes, could i get some more info on what they do?